packages/workspace/src/utils/object-utils.ts

```diff
@@ -0,0 +1,19 @@
+function isObject(v) {
+  return '[object Object]' === Object.prototype.toString.call(v);
+}
+
+export function sortObject(o: any) {
```

Comment:
This needed to be made so that the values of the projects in nx.json and workspace.json are hashed deterministically since order of properties and the order of the tags shouldn't cause a cache miss.